metisp/pymetis/src/pymetis/recipes/img/metis_lm_img_basic_reduce.py

```diff
@@ -153,7 +153,7 @@ def process_images(self) -> Dict[str, PipelineProduct]:
         return self.products
 
 
-class MetisLmBasicReduce(MetisRecipe):
+class MetisLmImgBasicReduce(MetisRecipe):
```

Comment:
The `_name` should also become `metis_lm_img_basic_reduce`, but we can also change that in another PR

Reply:
Fixed in #61